packages/pagebuilder/lib/ContentTypes/Block/__tests__/__snapshots__/block.spec.js.snap

```diff
@@ -23,11 +23,6 @@ exports[`renders a Block component 1`] = `
 >
   <div
     className="root"
-    dangerouslySetInnerHTML={
```

Comment:
Seems weird for this snapshot to be modified for moving the logic to another directory.